### PR TITLE
SPE10 Updated Documentation

### PR DESCRIPTION
**Pull request by OPMUSER**
Remove ODP file documentation and replace with Markdown documentation, contents are virtually the same.

Ready for review.

### CODE REVIEW

File: spe10/SPE10.md
Comment (bska) [inline spe10/SPE10.md:25]: Tiny nit: "... if THE THE case is ..."

**Review by bska — COMMENTED** [review]
It's probably not a change from earlier versions of this document, but what are the conditions on republishing simulation results of other vendors/simulators?

File: spe10/SPE10.md
Comment (OPMUSER) [reply spe10/SPE10.md:25]: @bska I don't know for sure, so I'll defer to someone wiser than me. Note that none of the plots are taken from the actual papers, the data was digitized and the plots in the documentation were derived from the digitized data. Since we state the source, then I don't think it is an issue. But like I said, I'll defer to someone wiser.

Will fix the typo.

**Comment by OPMUSER** [comment]
This has been sitting for a while, can it go in, or are there any change requests?

**Comment by bska** [comment]
> This has been sitting for a while, can it go in,

The PR is blocked by deciding whether or not it is okay to republish simulation results taken from the SPE paper.

**Comment by OPMUSER** [comment]
> The PR is blocked by deciding whether or not it is okay to republish simulation results taken from the SPE paper.

@bska Well we do this all the time, as for example in:
2022-11-29 17_39_18-The Open Porous Media Flow Rese - Atgeirr Flo Rasmussen pdf - Adobe Acrobat

Where we compare SPE1, SPE3, SPE5 ans SPE9. So I don't see an issue with this.

**Comment by alfbr** [comment]
My take on it is that if in doubt, we do not republish copyrighted material. Two wrongs do not make a right. As I see it, it is a learning process where we all do our best, and where we have stumbled in the past. I believe the easiest solution here is simply remove the plots that are copyrighted by others. One may link to them instead. Alternatively, somebody should dig up the reproduction requirements of the copyrighted material. Personally, I have spent way too much time reading through such documents, and hope to avoid it this time around.

File: spe10/SPE10.md
Comment (bska) [inline spe10/SPE10.md:15]: I guess if we want to be **really** picky it should be Heriot-Watt (two 't's) here.

**Review by bska — COMMENTED** [review]
Thanks a lot for the updates.  With one minor provision I'm prepared to merge this now.

File: spe10/SPE10.md
Comment (OPMUSER) [reply spe10/SPE10.md]: Done both in Markdown and decks.

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge this into master now.

**Pull request merged**
